app/views/candidate_interface/application_form/submit_success.html.erb

```diff
@@ -14,17 +14,17 @@
   <div class="govuk-grid-column-two-thirds">
     <h2 class="govuk-heading-m">What happens next</h2>
     <p class="govuk-body">
-      You will receive an email from the Apply for postgraduate teacher training service to confirm the successful submission of your application.
+      The Apply for teacher training service will send you an email to confirm the successful submission of your application.
```

Comment:
This feels like a change from active voice to passive.
Maybe "We have sent you"? They should get the email almost immediately.